operatorconfig/moduleconfig/authorization/v2.0.0-alpha/deployment.yaml

```diff
@@ -705,27 +705,27 @@ spec:
                   if [ "$ROLE" = "master" ]; then
                       MASTER=$i.authorization.svc.cluster.local
                       echo "Master found at $MASTER..."
-                      foundMaster=$true
+                      foundMaster=true
                       break
                   else
                     MASTER=$(redis-cli --no-auth-warning --raw -h $i -a $REDIS_PASSWORD info replication | awk '{print $1}' | grep master_host: | cut -d ":" -f2)
                     if [ "$MASTER" = "" ]; then
                         echo "Master not found..."
-                        echo "Sleeping 5 seconds for redis pods to come up..."
+                        echo "Waiting 5 seconds for redis pods to come up..."
```

Comment:
your helm charts PR is using "Sleeping", can we use the same wording here and in helm charts?